composer.json

```diff
@@ -21,11 +21,13 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "illuminate/auth": "^10.0|^11.0",
-        "illuminate/container": "^10.0|^11.0",
-        "illuminate/contracts": "^10.0|^11.0",
-        "illuminate/database": "^10.0|^11.0",
-        "laravel/serializable-closure": "^1.0|^2.0"
+        "illuminate/auth": "^10.0|^11.0|^12.0",
+        "illuminate/container": "^10.0|^11.0|^12.0",
+        "illuminate/contracts": "^10.0|^11.0|^12.0",
+        "illuminate/database": "^10.0|^11.0|^12.0",
+        "laravel/serializable-closure": "^1.0|^2.0",
+        "laravel/framework": "12.*",
```

Comment:
Why is there now a dependency on `laravel/framework` in addition to the `illuminate/*` dependencies?

Requiring `12.*` prevents installation on Laravel 10 and 11, even though these are possibly by the `illuminate/*` constraints and are tested against in `.github/workflows/tests.yml`.

Reply:
Wow, this is a serious error, I don't know how I missed it. Thanks for the comment.